opensrp-app/src/main/java/org/smartregister/repository/TaskRepository.java

```diff
@@ -112,6 +121,18 @@ public static void createTable(SQLiteDatabase database) {
         database.execSQL(CREATE_TASK_PLAN_GROUP_INDEX);
     }
 
+    /**
+     * Migrate the older database by add restriction columns and changing of priority to enum
+     *
+     * @param database the database being upgraded
+     */
+    public static void updatePriorityToEnumAndAddRestrictions(@NonNull SQLiteDatabase database) {
+        DatabaseMigrationUtils.addColumnIfNotExists(database, TASK_TABLE, RESTRICTION_REPEAT, VARCHAR);
+        DatabaseMigrationUtils.addColumnIfNotExists(database, TASK_TABLE, RESTRICTION_START, VARCHAR);
+        DatabaseMigrationUtils.addColumnIfNotExists(database, TASK_TABLE, RESTRICTION_END, VARCHAR);
```

Comment:
Drop priority column and readd it as varchar column type

Reply:
I have updated this to Integer

Reply:
Migration of priority column since on running apps its an integer

Reply:
That is is being done as explained https://github.com/OpenSRP/opensrp-client-core/pull/691#discussion_r527628418